encoding/x/protobuf/types.go

```diff
@@ -33,7 +33,7 @@ import (
 
 const (
 	// Encoding is the name of this encoding.
-	Encoding transport.Encoding = "protobuf"
+	Encoding transport.Encoding = "proto"
```

Comment:
If this is what we want going forward we should validate that no one is using this an release a patch ASAP.  Otherwise we need to support registering the procedure twice under both encodings.